core/src/main/scala/fs2/io/ssh/Process.scala

```diff
@@ -86,16 +81,16 @@ final class Process[F[_]: Concurrent: ContextShift] private[ssh] (
   private[this] def ioosToSink(ioosF: F[IoOutputStream]): Pipe[F, Byte, Unit] = { in =>
     Stream.eval(ioosF) flatMap { ioos =>
       val written = in.chunks evalMap { chunk =>
-        val bytes = chunk.toBytes
-        val buffer = new ByteArrayBuffer(bytes.values, bytes.offset, bytes.length)
+        val bytes = chunk.toArray
+        val buffer = new ByteArrayBuffer(bytes, 0, bytes.length)
         buffer.wpos(bytes.length)
```

Comment:
This line is no longer necessary (it is actually incorrect and should have been `bytes.offset + bytes.length`) as the [`ByteArrayBuffer(byte[], Int, Int)` constructor](https://github.com/apache/mina-sshd/blob/sshd-2.9.2/sshd-common/src/main/java/org/apache/sshd/common/util/buffer/ByteArrayBuffer.java#L102) already handles this.

Reply:
Removed 👍